### ISSUE TITLE
\Twig\Environment::createTemplate will return \Twig\TemplateWrapper, not \Twig_Template

### ISSUE THREAD

**Issue by peter279k** [wdes/php-I18n-L10n#3]
As title, the Twig will change return instance in this `\Twig\Environment::createTemplate` method when Twig version is `v2.11.0`.

The failed Travis CI build log is available [here](https://travis-ci.org/peter279k/php-I18n-L10n/jobs/541177924).

The previous Travis CI build log is fine because the package used  the `Twig` version is `v2.6.0`.

And it also effects the `preg_match_all` behavior inside `MemoryCache::extractDoDisplayFromCache` method.

## References
- [Twig\Environment::createTemplate in Twig v2.6.0](https://github.com/twigphp/Twig/blob/v2.6.0/lib/Twig/Environment.php#L417).
- [Twig\Environment::createTemplate in Twig v2.11.0](https://github.com/twigphp/Twig/blob/v2.11.0/lib/Twig/Environment.php#L417)

**Comment by williamdes** [comment]
Hi @peter279k This should be fixed by e1803bdc4af0f96e1e319d6615ff487c20fe3db7
I will release 1.0.0 soon
Thanks for noticing this issue !

**Comment by peter279k** [comment]
Hi @williamdes, thank you for your concern.

It seems that latest Travis CI build log is failed due to following fatal error message during PHPUnit tests:

```bash
TypeError: Argument 1 passed to Wdes\PIL\Twig\MemoryCache::extractDoDisplayFromCache() must be an instance of Twig\Template, instance of Twig\TemplateWrapper given
```

**Comment by peter279k** [comment]
The latest commit looks fine in `dev` branch.

Closed.

**Comment by williamdes** [comment]
@peter279k I released version 1.0.0 :tada: 
I fixed some bugs and updated the README.md ;)
